sdk/analytics/src/main/java/com/klaviyo/analytics/Klaviyo.kt

```diff
@@ -26,6 +29,11 @@ import com.klaviyo.core.safeCall
  */
 object Klaviyo {
 
+    /**
+     * Queue of failed operations attempted prior to [initialize]
+     */
+    private val preInitQueue: Queue<Operation<Unit>> = LinkedList()
```

Comment:
I may be misunderstanding here but are we using this both as a pre init queue as well as an error queue? I saw that when this is passed to exception handling it was called error queue? 

I ask mainly because should this name reflect that and also just trying to understand what exceptions would cause requests added to this queue?

Reply:
Yeah it's a little convoluted but the exceptions we catch with safeCall/safeApply are due to not initializing, so the queue gets added to when handlepush is called pre-initialize. 
This was originally a more generic solution that's why I did it that way

Reply:
So basically, yes it is an error queue, but just for pre-init errors.